Lib/test/test_ntpath.py

```diff
@@ -1095,12 +1242,11 @@ def test_isjunction(self):
                 self.assertFalse(ntpath.isjunction('tmpdir'))
                 self.assertPathEqual(ntpath.realpath('testjunc'), ntpath.realpath('tmpdir'))
 
-    @unittest.skipIf(sys.platform != 'win32', "drive letters are a windows concept")
-    def test_isfile_driveletter(self):
-        drive = os.environ.get('SystemDrive')
-        if drive is None or len(drive) != 2 or drive[1] != ':':
-            raise unittest.SkipTest('SystemDrive is not defined or malformed')
-        self.assertFalse(os.path.isfile('\\\\.\\' + drive))
+    def test_isfile_invalid_paths(self):
+        self.assertIs(ntpath.isfile('/tmp\udfffabcds'), False)
+        self.assertIs(ntpath.isfile(b'/tmp\xffabcds'), False)
+        self.assertIs(ntpath.isfile('/tmp\x00abcds'), False)
+        self.assertIs(ntpath.isfile(b'/tmp\x00abcds'), False)
 
```

Comment:
Why do you remove this test?

Reply:
This is a mistake. Thank you for noticing.